config/routes.rb

```diff
@@ -1,3 +1,17 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  namespace :v1, defaults: { format: 'json' } do
+    post 'auth/login', to: 'authentication#authenticate'
+    post 'signup', to: 'users#create'
+
+    resources 'restaurants', only: [ :index, :show ]
+
+    resources :reservations
```

Comment:
- [ ] Don't forget to specify which routes you are really using here, we don't want to create empty routes.